### DISCUSSION TITLE
Tree Component

### DISCUSSION THREAD

**Discussion by kquadri** [tusen-ai/naive-ui#2775] [Q&A]
Hi, 
In Tree Component, if there are larger number of children node, then it performs like this.

![Screenshot from 2022-04-12 11-09-26](https://user-images.githubusercontent.com/76086557/162888930-b07e8c5b-1c04-45a4-b045-10d550a38dea.png)

how can I fix this issue ?

**Comment by 07akioni** [comment]
I think we should provide another mode to solve it.
